finetune/target_models/masked_language_model.py

```diff
@@ -99,6 +103,9 @@ class MaskedLanguageModel(BaseModel):
     :param config: A :py:class:`finetune.config.Settings` object or None (for default config).
     :param \**kwargs: key-value pairs of config items to override.
     """
+
+    defaults = dict(low_memory_mode=True, batch_size=48, xla=True)
```

Comment:
you've found that these are better than the config.py defaults in terms of speed?